### DISCUSSION TITLE
How do I measure for direct drive?

### DISCUSSION THREAD

**Discussion by kyzitemelos** [Jyers/Marlin#327] [Q&A] [answered]
I have a micro swiss direct drive and hotend but am unable to measure for it and have followed Micro Swiss's vague instructions but UBL is still not lining up.

**Comment by Jyers** [comment, marked as answer]
The best method I have found to measure nozzle to probe offsets is to move the nozzle to any point from the move menu. Mark that point on the bed, then move the printhead until the probe is directly over that point. The differences between these two positions are your offsets
